developer-docs/docker.md

```diff
@@ -34,9 +34,15 @@ Once the Puppetserver is ready, the following message is reported:
 2019-03-18 15:42:19,965 INFO  [p.s.l.legacy-routes-service] The legacy routing service has successfully started and is now ready to handle requests
 ```
 
+At this point it is fine to generate certs for the `aceserver`, this can be achieved though:
```

Comment:
is it "fine", or is it "required to get your development setup running"?

Reply:
:-D I'll fix the wording